.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,56 @@
+name: Bug Report
+description: File a bug or issue
+title: "bug: "
+labels: [bug]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        **Before** reporting an issue, make sure to search [existing issues](https://github.com/tree-sitter/tree-sitter-PARSER_NAME/issues). Usage questions such as ***"How do I...?"*** belong in our [Matrix room](https://matrix.to/#/#tree-sitter-chat:matrix.org) and will be closed.
```

Comment:
Maybe add the link to [`config.yml`](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#configuring-the-template-chooser) instead of here.